modulus/metrics/diffusion/loss.py

```diff
@@ -445,6 +445,9 @@ def __init__(
         P_mean: float = 0.0,
         P_std: float = 1.2,
         sigma_data: float = 0.5,
+        pos_embed: int = 4,
+        hr_mean_conditioning: bool = False,
+        gridtype: str = "sinusoidal",
```

Comment:
Why does the loss function need to know about this? Seems these things could all be encapsulated in the network.